plugins/base/src/main/kotlin/parsers/MarkdownParser.kt

```diff
@@ -48,9 +48,18 @@ class MarkdownParser(
         private fun emphasisHandler(node: ASTNode): DocTag =
             DocTagsFromIElementFactory.getInstance(
                 node.type,
-                children = listOf(visitNode(node.children[node.children.size / 2]))
+                children = node.children.evaluateChildrenWithDroppedEnclosingTokens(1)
             )
 
+        private fun strongHandler(node: ASTNode): DocTag =
+            DocTagsFromIElementFactory.getInstance(
+                node.type,
+                children = node.children.evaluateChildrenWithDroppedEnclosingTokens(2)
+            )
+
+        private fun List<ASTNode>.evaluateChildrenWithDroppedEnclosingTokens(count: Int) =
+            drop(count).dropLast(count).evaluateChildren()
```

Comment:
I'd personally just create a method only for dropping *count* first and last tokens and call `evaluateChildren()` explicitly (I mean like: `node.children.dropSurrounding(1).evaluateChildren()`) but this is also OK I think